File: Java/1071.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner input = new Scanner(System.in);
        
        int X = input.nextInt();
        int Y = input.nextInt();
        int tot = 0;

        if (X > Y) {
            for (int i = X - 1; i > Y; i--) {
                if (i % 2 != 0) {
                    tot += i;
                }
            }
        } 
        else {
            for (int i = Y - 1; i > X; i--) {
                if (i % 2 != 0) {
                    tot += i;
                }
            }
        }

        System.out.println(tot);
    }
 
}

File: Python/1071.py
# -*- coding: utf-8 -*-
x = int(input())
y = int(input())

tot = 0

if(x > y):
    t = x
    x = y
    y = t

if(x % 2 != 0):
    x += 2
    while(x < y):
        if x % 2 != 0:
            tot += x
        x += 1
else:
    x += 1
    while(x < y):
        if x % 2 != 0:
            tot += x
        x += 1

print(tot)